fluent-bit/examples/override-fluentbit-http.yaml

```diff
@@ -0,0 +1,74 @@
+---
+#override-fluentbit-http.yaml
+fluent-bit:
+  #endpoint: <Coralogix_endpoint> --> Only if different from the defaults
```

Comment:
Why do we override the entire file?

Reply:
if the appname and subname are static, you must override the 'copy' under the filter section, in order to do it you must include the config, and the filters section